plugins/poll/assets/stylesheets/common/poll-breakdown.scss

```diff
@@ -0,0 +1,116 @@
+.poll-breakdown-modal {
```

Comment:
do we use `rem` elsewhere in the app ?

Reply:
Yeah, here and there. But it's not used as much as I'd like to. 😉

I prefer `rem` over `em` for use in components because it better isolates them from the context they're being embedded into. For example, if I add a share button to some paragraph that happens to use a smaller font size, I wouldn't expect it to be smaller but rather exactly the same as anywhere else in the app.

Reply:
I found 11 places for now, so rather not used :p

We should talk of this with @awesomerobot, I generally prefer rem too and wish we would use it more, but... :p

Reply:
It might be 11 now but with this PR the number will soar to 29! 😂